scipy/sparse/__init__.py

```diff
@@ -84,53 +77,77 @@
    random_array - Random values in a given shape array
    block_array - Build a sparse array from sub-blocks
 
-Building sparse matrices:
+.. _stacking-arrays:
+
+Stacking arrays
```

Comment:
I wouldn't call this "stacking". Typically that term is reserved for `*stack` functions, and not for things like `kron` or `block_diag`. The existing text for this is more appropriate I think.

Reply:
I changed "stacking" to "combining".